recipes/twitch-native-ipc/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
I do not see anything I know of which requires 3.12, can this please be confirmed?